sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -139,6 +139,35 @@ pub trait OakNode {
     fn invoke(&mut self, method: &str, req: &[u8], writer: ChannelResponseWriter);
 }
 
+impl<T: OakNode> crate::Node<Invocation> for T {
+    /// Perform a gRPC event loop for an [`OakNode`].
```

Comment:
nit: description not entirely accurate (there's no loop here)

Reply:
Done.